cfgmgr/intfmgr.cpp

```diff
@@ -420,7 +454,22 @@ bool IntfMgr::doIntfGeneralTask(const vector<string>& keys,
             FieldValueTuple fvTuple("mac_addr", MacAddress().to_string());
             data.push_back(fvTuple);
         }
-        
+
+        if (!proxy_arp.empty() || !vrf_name.compare(0, strlen(VNET_PREFIX), VNET_PREFIX))
```

Comment:
A small request for change : After some internal discussion, we don't want to have proxy_arp enabled explicitly for Vnets. Let that also be triggered by config_db entry. I'll make necessary modifications to requirement doc. Please remove this check on Vnets